site/static/examples/kind-with-registry.sh

```diff
@@ -13,14 +10,21 @@ if [ "${running}" != 'true' ]; then
     -d --restart=always -p "${reg_port}:5000" --name "${reg_name}" \
     registry:2
 fi
-reg_ip="$(docker inspect -f '{{.NetworkSettings.IPAddress}}' "${reg_name}")"
 
 # create a cluster with the local registry enabled in containerd
-cat <<EOF | kind create cluster --name "${KIND_CLUSTER_NAME}" --config=-
+cat <<EOF | kind create cluster --config=-
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
-containerdConfigPatches: 
+containerdConfigPatches:
 - |-
   [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:${reg_port}"]
-    endpoint = ["http://${reg_ip}:${reg_port}"]
+    endpoint = ["http://${reg_name}:${reg_port}"]
 EOF
+
+# connect the registry to the cluster network
+docker network connect "kind" "${reg_name}"
+
+# tell https://tilt.dev to use the registry
+for node in $(kind get nodes); do
+  kubectl annotate node "${node}" "tilt.dev/registry=localhost:${reg_port}";
```

Comment:
@nicks is this documented anywhere?

Reply:
nope, I'm hoping to promote a shared standard that is not tilt-specific, this is a hack to unblock people

Reply:
Awesome! 👍

Reply:
would be equally happy with an annotation like `kind.sigs.k8s.io/registry=localhost:${reg_port}` if you're uncomfortable with the tilt.dev annotation

Reply:
that sounds good to me :+1: 
I'm fine with tilt.dev but it probably looks better to use a kind annotation

Reply:
added some documentation
https://docs.tilt.dev/choosing_clusters.html#discovering-the-registry
that you can point people to if anyone asks

Reply:
i'm going to leave this for now since it already works and doesn't seem all that "offensive" in a demo script until I can think of how we want to be doing this in the future when we actually integrate something properly which is probably soonish anyhow.

Reply:
thanks for documenting this!

Reply:
is there somewhere we can follow the standardization effort? I think this is a really great idea

Reply:
nope, it's been mostly off-the-cuff conversations I've been having with people. It falls at the intersection of a few concerns (DevEx teams and cluster providers and image-build tools).

But this conversation is the kick in the pants I probably needed to start some more formal conversations. I'll kick that off by filing some issues in this repo + the k3d repo + the microk8s repo and see where that goes.